packages/server-wallet/src/wallet/store.ts

```diff
@@ -54,7 +54,12 @@ import {recoverAddress} from '../utilities/signatures';
 import {Outgoing} from '../protocols/actions';
 import {ObjectiveModel, DBObjective} from '../models/objective';
 import {logger} from '../logger';
+<<<<<<< HEAD
```

Comment:
tsk tsk tsk

Reply:
👀 I'll resolve this after we get a ✅ on this and before rebase & merge to `master`